modules/setting/setting.go

```diff
@@ -561,6 +561,15 @@ var (
 		TrustedFacets []string
 	}{}
 
+	// Metrics settings
+	Metrics = struct {
```

Comment:
Are you able to add default values to `custom/conf/app.ini.sample` and also to `docs/content/doc/advanced/config-cheat-sheet.en-us.md`